README.md

```diff
@@ -6,19 +6,30 @@ relates to weekly certification
 ## Running the Application
 
 **Prerequisites:**
-  - Node 12+
-  - yarn
+
```

Comment:
Please provide feedback if you have thoughts about how to structure this info in the README. Very open to alternate documentation suggestions.

Reply:
CC @lomky

Reply:
I'm confused-- are you removing these because they aren't prerequisites?...

Reply:
Oh I see, the github preview made it look like these lines were being removed, but they're actually just being de-dented.

Reply:
Added docs look good, thanks!

Reply:
Looks good to me - I think the README is still short enough that the flat structure works but I'm open to more structure if it gets out of hand